### DISCUSSION TITLE
Some items do not work

### DISCUSSION THREAD

**Discussion by smoothchat** [calibx/msfsdeck#203] [General]
Just a couple of observations with v2.3.0.

Aircraft = C172 Classic

Parking Brakes - the indicator works (when toggled via the sim), however pressing the button appears to do nothing.

Pedestral Light - doesn’t appear to work.
CabinLight - doesn’t appear to work either. I think these require a “SET” event rather than a value change.

Instrument Lights - works, but you need to set Light_Potentiometer_3 to 100 otherwise, if the lights are turned down, they don’t illuminate on the button press

Thanks for making this excellent plugin available.

**Comment by calibx** [comment]
Parking brake is fixed in 2.4.0.

The lights depends of the aircarft ingame implementation : with the 747 all seems to be ok
